\documentclass{article}
\usepackage{amsmath, amssymb}
\usepackage{dirtytalk}
\usepackage{enumitem}
\usepackage{hyperref}    % Links in TOC
\usepackage[a4paper,bindingoffset=0.2in,%
             left=0.75in,right=0.75in,top=0.75in,bottom=0.75in,%
             footskip=.25in]{geometry}
\newenvironment{QandA}{\begin{enumerate}[label=\bfseries Q\arabic*.]}
                       {\end{enumerate}}
\newenvironment{answered}{\par\normalfont\underline{Answer:}}{}

\title{All Questions}
\author{Vince Reuter}
\date{Last updated Tuesday, August 18, 2020}

\begin{document}

\maketitle

\tableofcontents

\section{Evolutionary Classifications}
\begin{QandA}
  \item{What are the 3 clades of extant mammals?}
    \begin{answered}
    \begin{itemize}
      \item{Monotremes}
      \item{\textit{Metatheria} (marsupials)}
      \item{\textit{Eutheria} (\textit{placentalia})}
    \end{itemize}
    \end{answered}
  \item{What \textbf{anatomical feature} allows particularly \textit{long gestation} in one of the clades? How?}
    \begin{answered}
    The lack of an \textit{\textbf{epipubic bone} in Eutherians} allows abdominal expansion during pregnancy, in turn allowing longer gestation.
    \end{answered}
  \item{What are the extant species of \textbf{monotremes}?}
    \begin{answered}
    \begin{itemize}
      \item{Platypus}
      \item{4 different kinds of Echidna}
    \end{itemize}
    \end{answered}
  \item{Where are the \textbf{monotremes} found?}
    \begin{answered}
    All 5 extant monotremes are in \textbf{Australia}.
    \end{answered}
\end{QandA}
\section{Telomeres}
\begin{QandA}
  \item{\textbf{Telomere binding proteins (TBPs):} which protein is primarily responsible for \textit{preventing end-to-end fusions} of chromosomes? How so?}
    \begin{answered}
    \textbf{\textit{TRF2}} prevents end-to-end fusions (NHEJ) by preventing tetramerization (?) of \textit{Ku} proteins (spec., \textit{Ku70} and \textit{Ku80}) that would normally lead to NHEJ.
    \end{answered}
\end{QandA}
\section{Nucleosomes}
\begin{QandA}
  \item{What fundamental properties of histones that set them apart from most proteins account for excellent performance of function as DNA packaging proteins?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Size:} each of the histone proteins is quite small, useful for a functional role in a tight space like the nucleus, and for \textit{finer resolution} positioning}
      \item{\textbf{Charge/pH:} DNA is negatively charged and generally acidic (yielding protons in solution); histones have many \textit{basic} AA side chains (among highest combined K and R proportion of all proteins), strongly electrostatically attracting DNA.}
    \end{itemize}
    \end{answered}
  \item{What property of histones across species suggests their centrality of function to eukaryotic life?}
    \begin{answered}
    The histones are among the most highly conserved proteins, typically differing by just $2-5$ amino acids.
    \end{answered}
  \item{What structural feature of histones is both functionally essential and conserved in their evolutionary history?}
    \begin{answered}
    The \textit{histone fold} domain is essential for the \say{handshake} by which the octamer forms, and it's the domain that's shared by the nucleosomal histones.
    \end{answered}
\end{QandA}
\section{Balancer Chromosomes}
\begin{QandA}
  \item{What's a \textbf{\textit{balancer} chromosome?}}
    \begin{answered}
    A balancer chromosome is an artificial/synthetic/genetically engineered construct that features a number of structural variants designed to \textit{inhibit recombination.}
    \end{answered}
  \item{What are 3 key properties of a balancer chromosome?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Prevention of crossing over,} allowing a line with known mutations to be maintained without needing to constantly screen for the desired mutations}
      \item{\textbf{Homozygous lethality:} a balancer carries some recessive lethal allele, such that homozygosity is lethal (i.e., any balancer carrier is heterozygous.)}
      \item{\textbf{Dominant \textit{marker(s):}} a good balancer carries marker gene(s) that facilitate visual identification and/or selection, e.g. via poison}
    \end{itemize}
    \end{answered}
  \item{Why do balancer chromosomes accumulate so many variants, particularly deletions and large structural variants o/w likely to be harmful?}
    \begin{answered}
    Since they're \textit{only found as heterozygotes,} balancer chromosomes are subject to \textit{much \textbf{weaker negative selection}} against harmful variants.
    \end{answered}
  \item{How does a balancer chromosome prevent recombination (crossover suppression, i.e. preclude meiotic synapsis)?}
    \begin{answered}
    The inversions tend to prevent recognition of the homologous chromosome relationship.
    \end{answered}
  \item{What if, despite the inversions' tendency to suppress recombination, a balancer chromosome recombines with a homolog?}
    \begin{answered}
    The structural aberrations that will result from recombination between a balancer and its homolog should \textit{confer \textbf{lethality,}} preventing a population that should be heterozygous from coming to include homozyogtes.
    \end{answered}
\end{QandA}
\section{Experimental Methods General}
\begin{QandA}
  \item{What are the two main wins of \textit{in situ} Hi-C compared to standard (dilution) Hi-C?}
    \begin{answered}
    
    \end{answered}
  \item{What's the meaning of the \textit{in situ} portion of \textit{in situ} Hi-C? How's it differ from dilution Hi-C?}
    \begin{answered}
    The \textit{in situ} potion of \textit{in situ} Hi-C means that 
    \end{answered}
  \item{How does \textit{in situ} Hi-C attain its main wins relative to dilution Hi-C?}
    \begin{answered}
    \begin{itemize}
      \item{For greater resolution, \textit{in situ} Hi-C (at least the Rao/Lieberman-Aiden original from 2014) deployed a 4-cutter rather than a 6-cutter restriction enzyme.}
      \item{The \textit{in situ} aspect of \textit{in situ} Hi-C actually appears to reduce artifactual ligation events that can occur in dilution Hi-C. }
    \end{itemize}
    \end{answered}
  \item{Why/how does \textit{hydroxyurea} stall DNA replication}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Ribonucleotide reductase (RNR)} is an enzyme that reduces ribose (specifically, the 2' carbon) to deoxyribose by removing the hydroxyl group from 2' carbon.}
      \item{Hydroxyurea inhibits the catalytic activity of RNR, limiting the pool of dNTPs (DNA replication raw materials) in the cell.}
    \end{itemize}
    \end{answered}
  \item{Why may PCR on extract from cell culture or from a genetically engineered animal, after conditionally deleting an allele, still have \textit{some} minor signal from the deletion target?}
    \begin{answered}
    These are \textit{populations} of cells (bulk) from which the material for PCR is being taken, and \textit{recombination will likely be incomplete/nonuniversal.}
    \end{answered}
  \item{Why use a method like 3/4/5-C over Hi-C? What's an additional requirement, especially for 3/4C?}
    \begin{answered}
    \begin{itemize}
      \item{The \say{lower} C-based methods provide \textit{greater granularity/resolution} relative to Hi-C.}
      \item{There's an inherent cost savings that only having a restricted regional interest (i.e., not genome-wide) allows to leverage, since we need not sequence so much.}
      \item{The methods require specific regional primer design, and to have restricted interest/focus to one or a few loci.}
    \end{itemize}
    \end{answered}
  \item{\textbf{\textit{Capture-C} vs. \textit{traditional 3C}} What improvement does Capture-C make relative to traditional 3C?}
    \begin{answered}
    Capture-C adds \textit{biotinylation-based pulldown} for better purification of the library (?)
    \end{answered}
\end{QandA}
\section{Body Sections Axes Planes}
\begin{QandA}
  \item{What are the 3 main \say{slices,} or \say{sections,} of tissue that may be taken?}
    \begin{answered}
    \begin{itemize}
      \item{\textit{Coronal/frontal:} an instance of a longitudinal plane, the coronal plane runs parallel to the long body axis, perpendicular to the transverse plane, dividing dorsal from ventral}
      \item{\textit{: }}
      \item{\textit{Sagittal/longitudinal/median: }}
    \end{itemize}
    \end{answered}
  \item{Which body plane does the coronal plane intersect as a vertical line?}
    \begin{answered}
    The coronal plane intersects the sagittal plane as a vertical line.
    \end{answered}
  \item{Which body plane does the coronal plane intersect as a horizontal line?}
    \begin{answered}
    The coronal plane intersects the transverse plane as a horizontal line.
    \end{answered}
  \item{Which two body poles does the coronal plane separate?}
    \begin{answered}
    The coronal plane separates ventral from dorsal.
    \end{answered}
  \item{Which body plane does the sagittal plane intersect as a vertical line?}
    \begin{answered}
    The sagittal plane intersects the coronal plane as a vertical line.
    \end{answered}
  \item{Which body plane does the sagittal plane intersect as a horizontal line?}
    \begin{answered}
    The sagittal plane intersects the transverse plane as a horizontal line.
    \end{answered}
  \item{Which body poles does the sagittal plane divide/define?}
    \begin{answered}
    The sagittal plane defines/divides the left and right. Any non-median sagittal plane implies a lateral vs. medial division.
    \end{answered}
  \item{To which two body planes is the transverse plane perpendicular}
    \begin{answered}
    A transverse plane is perpendicular to both a sagittal plane and a coronal plane
    \end{answered}
  \item{Which two body poles does a transverse plane define/divide?}
    \begin{answered}
    A transverse plane divides anterior from posterior.
    \end{answered}
\end{QandA}
\section{Bioinformatics File Formats}
\begin{QandA}
  \item{Descibe the BED coordinate system.}
    \begin{answered}
    BED uses 0-based inclusive starts and exclusive ends, so a half-open interval; the first nucleotide has 0 start and 1 end.
    \end{answered}
  \item{What justifies the (perhaps quirky) BED coordinate system choice?}
    \begin{answered}
    Computation time justifies BED coordinates; specifically, length of a region/feature becomes simple subtraction of start from end.
    \end{answered}
\end{QandA}
\section{Analytical Methods}
\begin{QandA}
  \item{In which kind of graph/plot does a \textbf{power law} reveal itself as a \textbf{linear} relationship? Why?}
    \begin{answered}
    \begin{itemize}
      \item{A \textbf{power law} relation between variables is revealed as a \textit{linear} visualization in a \textbf{log-log plot}.}
      \item{$y = b x^a \implies \operatorname{log}(y) = a \operatorname{log}(x) + b$, so taking logs gives a linear equation relating log-response to log-input, with power as slope.}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{Nucleic Acid Protein Interactions}
\begin{QandA}
  \item{What biochemical property would be desirable for the 3D amino acid region of a protein that must bind a nucleic acid? In other words, what amino acid trait might we expect to be enriched in a nucleic acid binding region of a protein? Why?}
    \begin{answered}
    Since nucleic acids are negatively charged, higher affinity binding between the protein and the nucleic acid will be achieved if the protein's amino acids in that region (again, perhaps discontinuous / not contiguous in linear amino acid sequence) are \textbf{\textit{positively} charged.} Note, however, that while higher binding affinity may generally be preferred, that's very likely not \textit{always} the case.
    \end{answered}
  \item{What properties together of a plasma membrane and amino acids in a protein make for good binding?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{\textit{Negatively} charged cytoplasmic leaflet} of the plasma membrane, due to \textit{flippases'} regulatory action}
      \item{\textbf{\textit{Positively} charged amino acid} subsequence or folded region of a protein}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{Classic Experiments}
\begin{QandA}
  \item{Which experiment first to suggest that \textit{genetic material may be \textbf{horizontally} transmitted}?}
    \begin{answered}
    \textbf{Griffith's experiment} was the first to suggest the possibility of horizontal transfer of genetic material.
    \end{answered}
  \item{When was \textbf{Griffith's experiment}?}
    \begin{answered}
    1928
    \end{answered}
  \item{What did \textbf{Griffith's experiment} establish (i.e., what was the main finding)?}
    \begin{answered}
    Griffith's experiment established that \textit{bacteria can \textbf{transfer genetic material}} horizontally.
    \end{answered}
  \item{How was Griffith's experiment able to show what it did?}
    \begin{answered}
    Mixing extract heat-killed, virulent bacteria with live, benign bacteria made the previously benign bacteria pathogenic.
    \end{answered}
  \item{What was the process of horizontal gene transfer initially called (in Griffith's experiment)?}
    \begin{answered}
    The process now known as HGT was called \textit{transformation}, and the material was called the \textit{transforming principle}.
    \end{answered}
  \item{Which experiment, and when, \textit{solidified evidence for \textbf{DNA as the transforming principle}}?}
    \begin{answered}
    Results published in \textbf{1944} by \textit{Avery, MacLeod, and McCarty} supported DNA (not bacteria) as the transforming principle.
    \end{answered}
  \item{Why was it not thought possible for DNA to be the material of the genetic code? What was the alternative?}
    \begin{answered}
    DNA's \textit{biochemical boringness} was oft cited as reason that it couldn't encode life's complexity; \textit{protein} was the alternative.
    \end{answered}
  \item{How did \textit{Avery, MacLeod, and McCarty's} work elaborate Griffith's work to demonstrate \textit{DNA as the \textbf{transforming principle}}?}
    \begin{answered}
    Avery, MacLeod, and McCarty subjected the extract to a \textit{variety of \textbf{digestive enzymes}}, and the only one that could break down the extract was a deoxyribonuclease (not various proteases and ribonucleases).
    \end{answered}
  \item{Which classic experiment (and when) \textit{affirmed AMM}, demonstrating that \textbf{DNA is the transforming principle}?}
    \begin{answered}
    The \textbf{Hershey-Chase experiment} of 1952 affirmed Avery, MacLeod, and McCarty's work, showing that DNA is the transforming principle.
    \end{answered}
  \item{How did the \textbf{Hershey-Chase} experiment show what it did?}
    \begin{answered}
    \begin{itemize}
      \item{The Hershey-Chase experiment demonstrated that DNA is the transforming principle by \textit{radiolabeling specific elements}.}
      \item{Phosphorus is specific to DNA (not protein), and sulfur is specific, through cysteine, to proteins (not DNA).}
      \item{Phages incubated with either radiolabeled sulfur or radiolabeled phosphorus then specifically are marked for either protein or DNA, respectively.}
      \item{Radioactivity stays with the protein, entering the infected bacteria only when it's phosphorus that's labeled and staying with the phage otherwise.}
      \item{Progeny phages are labeled if and only if the parent phages were made with radiolabeled phosphorus, not DNA, showing that \textit{\textbf{DNA}, but not protein, is \textbf{heritable}}}
    \end{itemize}
    \end{answered}
  \item{What's the nickname for the Hershey-Chase experiment? Why?}
    \begin{answered}
    The Hershey-Chase experiment is nicknamed the \say{blender experiment} b/c of use of the blender to remove the phages from the bacteria.
    \end{answered}
  \item{Which experiment (and when) showed that \textit{\textbf{RNA} can be the material of \textbf{heredity}}?}
    \begin{answered}
    1955's \textbf{Fraenkel-Conrat} experiment showed that RNA can encode heredity.
    \end{answered}
  \item{Which virus was used to show that \textit{\textbf{RNA} can be the material of \textbf{heredity}}?}
    \begin{answered}
    Tobacco mosaic virus (TMV)
    \end{answered}
  \item{How was RNA shown to be capable of being heredity's material?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{TMV} in the 1955 \textbf{Fraenkel-Conrat experiment} was used in two strains, and DNA from one was mixed with protein of the other to make 2 kinds of chimeric virus particles.}
      \item{Progeny viruses had the protein coats matching that of the strain from which the parent's RNA was from (i.e., the chimeric state lasted only one generation and was not heritable.)}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{General Membrane Biology}
\begin{QandA}
  \item{In what important electrostatic way do animal/eukaryotic cell membranes differ from microbe membranes?}
    \begin{answered}
    \textbf{\textit{Charge} distribution}: microbes have fairly randomly distributed charge w.r.t. cytoplasmic vs. exoplasmic side. In contrast, \textit{animal cells tend to \textbf{organize}} the charge distribution.
    \end{answered}
  \item{How does the main electrostatic difference between eukaryotic cell membrane and microbe cell membrane relate to \textbf{defensins} of \textit{innate immunity}?}
    \begin{answered}
    \begin{itemize}
      \item{Defensins work basically by boring a hole in the plasma membrane, using positive charges.}
      \item{Microbes are vulnerable to defensins since they have some negative charge on the exoplasmic side, since their charge distribution is largely random.}
      \item{Animal/eukaryotic/host cells are protected from their own defensins because they tend to organize the negative charges of the lipid bilayer on the cytoplasmic membrane side, such that few, if any, are exposed exoplasmically to the defensins.}
    \end{itemize}
    \end{answered}
  \item{\textit{How} do animal/eukaryotic cells regulate the electrostatic properties of their membranes?}
    \begin{answered}
    \textbf{Flippases,} or \textit{phospholipid translocases}: these enzymes facilitate the \say{flipping} of a phospholipid from one face of the bilayer to the other, which would otherwise be energetically disfavored.
    \end{answered}
  \item{\textit{What molecules} (\textbf{phosphilipids,} specifically) are heavily involved in membrane biology? Name 3. Which in particular is involved in charge distribution?}
    \begin{answered}
    \begin{itemize}
      \item{Phosphatidyl\textit{serine}}
      \item{Phosphatidyl\textit{choline}}
      \item{Phosphatidyl\textit{ethanolamine}}
    \end{itemize}
    \end{answered}
  \item{Describe the basic biochemical structure of the 3 main phosphilipids mentioned in \textit{Molecular Biology of the Cell} ($566-567$)?}
    \begin{answered}
    \begin{itemize}
      \item{Distinguishing group (e.g., choline): this, the phosphate group, and glycerol jointly comprise the \textbf{\textit{hydrophilic} \say{head} group}}
      \item{\textbf{Phosphate} group: a sort of linker}
      \item{\textbf{Glycerol}: all three hydroxyl groups participate in dehydration synthesis / condensation reactions, with one each going to fatty acids and the third to the phosphate group.}
      \item{\textbf{Fatty acids}: a pair of fatty acid chains, \textit{one unsaturated}, comprise the \textbf{\textit{hydrophobic} \say{tail}}}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{DNA repair}
\begin{QandA}
  \item{What is \textit{DNA \textbf{resection?}}}
    \begin{answered}
    DNA resection is the \textit{exonucleolytic \say{chewback}} of bases, typically from a double-stranded break
    \end{answered}
  \item{In what processes is \textit{DNA \textbf{resection}} a key step/subprocess?}
    \begin{answered}
    \textbf{\textit{DNA repair}}, via \textit{homologous recombination (HR):} DNA resection is critical for the homologous recombination (HR), typically as part of the \textit{double-stranded break repair \textbf{(DSBR)}} response.
    \end{answered}
  \item{How does \textit{DNA \textbf{resection}} perform its function (facilitation of initiation of HR)?}
    \begin{answered}
    DNA resection \textit{creates \textbf{3' overhang,}} forming the ssDNA that will perform the \textbf{strand \textit{invasion}} step of HR, invading a nearby duplex region.
    \end{answered}
  \item{\textbf{\textit{Double} strand break repair (DSBR):} The balance between which 2 proteins is a primary determinant of the procession and pathway chosen for DSBR? How So?}
    \begin{answered}
    The balance between \textit{BRCA1} and \textit{TP53BP1} is a major determinant of the balance between candidate DBSR pathways, exerting great influence over which pathway is chosen.
    \end{answered}
  \item{What process does \textit{TP53BP1} disfavor, and how does that influence a cell's response to / pathway chosen for double-stranded break repair (DSBR)?}
    \begin{answered}
    \textit{TP53BP1} disfavors end resection. Since end \textit{resection promotes HR,} TP53BP1 tilts the balance of power away from HR and in favor of more error-prone NHEJ, which is worse for the cell.
    \end{answered}
  \item{Which process does \textit{BRCA1/2} favor, and how does that influence a cell's response to / pathway chosen for DSBR?}
    \begin{answered}
    \textit{BRCA1/2} associate with \textit{Rad51} to help initiate/facilitate/orchestrate \textbf{homologous recombination.} As such, cells deficient in those proteins will tend to more frequently use NHEJ to repair double-strand DNA break damage, leading to more costly and dangerous repair errors.
    \end{answered}
\end{QandA}
\section{HIV}
\begin{QandA}
  \item{Why is HIV so interesting to study for alternative splicing?}
    \begin{answered}
    \begin{itemize}
      \item{The entire HIV proteome comes from a \textbf{single \textit{primary transcript}}, so alternative splicing is an essential component of HIV gene expression.}
      \item{The structural conformation of HIV RNA plays a significant role in how it's spliced, so the virus provides an interesting perspective on the link between structure and splicing.}
    \end{itemize}
    \end{answered}
  \item{Which protein, made in the early phase, plays a major role in late phase and binds to a particular subsequence?}
    \begin{answered}
    The \textbf{Rev} protein is made early and binds Rev late, helping get gRNA out of the nucleus.
    \end{answered}
  \item{Roughly where in the HIV genome is the RRE?}
    \begin{answered}
    The RRE is near the 3' end of the genome.
    \end{answered}
  \item{What's the \textbf{RRE}?}
    \begin{answered}
    \textbf{Rev response element}: this is a regulatory sequence that the Rev protein binds to.
    \end{answered}
  \item{What process is the sole determinant of whether an mRNA will be able to code for the protein \textit{Tat}? What region of the HIV genomes is most relevant for this for Tat in partciular?}
    \begin{answered}
    \begin{itemize}
      \item{Alternative splicing entirely determines whether \textit{Tat} will be made; without the use of a particular splice site, Tat is not made.}
      \item{It's the \textbf{A3 splice site} of HIV that determines whether Tat will be made, and that (A3 SS) region of the RNA molecule exists in a couple different major structural conformations with differential accessibility of the site to the splicing machinery.}
    \end{itemize}
    \end{answered}
  \item{Which regions of the HIV genome are both especially divergent in structural conformation and have especially stable alternative conformations?}
    \begin{answered}
    Both the 5' UTR and the \textbf{Rev response element (RRE)} are have stable, divergent structures, as revealed by DMS-MaPseq and application of the DREEM algorithm.
    \end{answered}
  \item{What clever trick must HIV pull toward the end of its lifecycle?}
    \begin{answered}
    \begin{itemize}
      \item{The \textit{integrated \textbf{provirus}} (DNA) form of the genome is the template for RNA synthesis (i.e., HIV uses the \textit{reverse transcriptase} strategy rather than the RNA-dependent RNA polymerase (RdRP) as the biosynthetic machinery to account for host cells' lack of it.}
      \item{Since the \textbf{full-length \textit{primary} transcript} is not only mRNA for alternative splicing and protein synthesis, but also the viral genome, the virus \textit{must export \textbf{unspliced} RNA} from the nucleus.}
      \item{Getting unspliced RNA out of the nucleus is tricky becuase ordinarily the exportins that interact with nuclear pore complex proteins to license export are applied in the course of splicing; HIV must find an alternative.}
    \end{itemize}
    \end{answered}
  \item{What are \say{elite controllers}, and why are they interesting to study?}
    \begin{answered}
    \begin{itemize}
      \item{\say{Elite controllers} are HIV patients that \textit{keep virus down} at very low levels without the assistance of antiretroviral therapy (ART).}
      \item{\say{Elite controllers} are interesting to study for the insight they may bring to the disease biology and pathogenesis, as well in a broader sense for the connection between robustness/flexibility of the adaptive immune response to rapidly mutating pathogens.}
      \item{The known \textit{protective HLA allele} (HLA-B57) greatly enriched in these individuals is known to present a narrower range of host peptides to T cells during their development in the thymus, leading to a cognate TCR pool restricted to this allele's MHC molecule that tends to depend on fewer individual, but strong peptide contacts.}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{General}
\begin{QandA}
  \item{Why care about interactions between nucleic acids and proteins? Why are they important?}
    \begin{answered}
    \begin{itemize}
      \item{Events' \textbf{\textit{timing} specificity} (think TF concentration in \textbf{developmental \textit{stage}})}
      \item{Events' \textbf{\textit{spatial} specificity} (think TF concentration in \textbf{subsellular \textit{compartment}})}
    \end{itemize}
    \end{answered}
  \item{Describe/characterize the \textit{relative merits} / \say{tradeoff} of virtue between EMSA and footprinting methods.}
    \begin{answered}
    \textbf{\textit{Sensitivity} vs. \textit{specificity:}} EMSA is more sensitive (looser robustness requirement for probe binding by protein), but it provides a coarser picture / less resolution than footprinting for \textit{where} a protein of interest binds.
    \end{answered}
  \item{What's a key piece of information likely to be of interest that \textit{neither EMSA nor a footprinting method} provides? What class of method does?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Protein identity:} neither EMSA nor footprinting provides much information about the identity of protein binding.}
      \item{ChIP (chromatin immunoprecipitation) methods aim provide binding protein identity information.}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{ChIP}
\begin{QandA}
  \item{What are the 2 main ChIP kinds/strategies and how do they differ?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{\textit{Primer}-based ChIP} targets a specific, \textit{individual} region of interest.}
      \item{\textbf{\textit{Chip}- or \textit{Sequencing}-based CHIP} (ChIP-Chip and ChIP-seq, respectively) can examine an \textit{entire genome} for evidence of protein binding.}
    \end{itemize}
    \end{answered}
  \item{Briefly describe \textbf{ChIP-PCR.} How is it used relative to other ChIP methods? What're strengths/weaknesses relative to other ChIP methods?}
    \begin{answered}
    \begin{itemize}
      \item{As with any other chromatin immunoprecipitation method, the availability of an antibody specific for a protein, a complex, or a modification of interest is critical.}
      \item{DNA abundance is quantified and compared between an experimental and control sample group; experimental/treatment receives specific antibody while control receives nothing or a nonspecific antibody.}
      \item{\textbf{\textit{Scope} difference:} a key difference with other ChIP methods is in the scope of investigation; ChIP-PCR examines one or few loci while others can examine many.}
      \item{The \textit{scope limitation} arises from the \textit{need to design primers.} Primers targeting a control (expected unbound) locus and the region(s) of interest must be designed.}
      \item{Owing to limited scope, ChIP-PCR is typically deployed where the line of inquiry is either \textit{\textbf{locally specific} to a region,} or as a \textbf{cost-effective \textit{control}} for a method with broader scope.}
      \item{Strength : cost :: weakness : scope}
    \end{itemize}
    \end{answered}
  \item{What are the \textit{main \textbf{validation} questions} to address in a \textbf{ChIP-PCR} experiment?}
    \begin{answered}
    \begin{itemize}
      \item{Interest is to compare control amplicon vs. target amplicon in \textit{\textbf{input} DNA} (no/nonspecific antibody) for \textbf{\textit{primer bias,}} and to compare control amplicon vs. target in \textit{\textbf{treated} DNA} to assess \textbf{\textit{antibody efficacy.}}}
      \item{\textbf{\textit{Antibody} efficacy:} If the antibody effectively binds the protein/complex/modification of interest, the treatment samples' target region should be enriched (lower cycle threshold) relative to those samples' control region. This suggests that the antibody's sensitive and specific.}
      \item{\textbf{\textit{Primer} efficacy/bias:} targeting a \textbf{\textit{control} locus/amplicon} helps to \textit{establish a \textbf{baseline}} for amplification of the respective regions. Then we can \textit{compare the \textbf{change in CT}} for a given sample. This facilitates comparison of the target to control in the treated sample, which is ultimately what we're after.}
      \item{\underline{Example:}}
      \begin{enumerate}
        \item{In input DNA, both the control amplicon and the target amplicon show CT of 27.}
        \item{In the IP sample(s), the control amplicon has CT of 30 while treated (specific Ab) samples have CT of 27. This implies $8x = 2^{3}x$ enrichment of the target.}
        \item{We can use the raw difference in the CT between the regions in the treated sample because the baseline difference is $0 = 27 - 27$ between the amplicons in the input DNA.}
      \end{enumerate}
    \end{itemize}
    \end{answered}
  \item{What's \textbf{\textit{cycle threshold?}} What information does it provide?}
    \begin{answered}
    \begin{itemize}
      \item{The \textbf{cycle threshold} is the \textit{number of PCR cycles} needed to sufficiently amplify some region of interest for detection by whatever machine (spectrophotometer?)}
      \item{The CT is a \textit{measure of \textbf{abundance}} (of molecules). Each PCR cycle amplifies (in theory) each molecule 2x, so abundance is inversely exponentially proportional to the CT.}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{EMSA}
\begin{QandA}
  \item{What's EMSA stand for?}
    \begin{answered}
    Electophoretic mobility shift assay
    \end{answered}
  \item{What's EMSA measure directly?}
    \begin{answered}
    EMSA measures the \textbf{distance traveled} by a biomolecule or biomolecular process, though a polyacrylamide gel
    \end{answered}
  \item{Broadly speaking, how does PAGE/gel electophoresis work? How does it provide a readout of intermolecular binding?}
    \begin{answered}
    \begin{itemize}
      \item{1. Gel impedes movement as negatively charged molecules migrate away from negative source}
      \item{2. Larger molecules experience stronger resistance}
      \item{3. More binding means larger molecular complexes.}
      \item{4. $d = rt \land \text{ } r \propto \text{ molecular (complex) size}$}
    \end{itemize}
    \end{answered}
  \item{What biomolecular phenomenon does EMSA aim to measure?}
    \begin{answered}
    EMSA aims to measure binding between proteins and nucleic acids (and proteins-to-proteins). Most generally, EMSA/PAGE/shift assays measure molecular (complex) size
    \end{answered}
  \item{What two main kinds of labeling are used to detect complexes in a shift assay?}
    \begin{answered}
    \begin{itemize}
      \item{Fluorescent (fluorphore conjugation)}
      \item{Autoradiography (make one end of nucleic acid radioactive)}
    \end{itemize}
    \end{answered}
  \item{What main precaution must be taken when using a fluorophore to label molecules for detection in a shift assay?}
    \begin{answered}
    The fluorophore must not interfere with binding between the molecules of interest
    \end{answered}
  \item{In simplest form, EMSA provides information, e.g., \textbf{that \textit{a}} protein bound DNA, but \textbf{not \textit{which specific} protein(s)} bound DNA. How can that be assessed?}
    \begin{answered}
    \textbf{Antibody \textit{supershift.}} Antibody binding can be protein-specific, and thus will specifically decrease mobility of compatible protein(s), allowing identity inference.
    \end{answered}
  \item{How may the presence of multiple proteins simultaneously binding to DNA be examined, via shift assay?}
    \begin{answered}
    Each binding event will increase size and weight of the molecular complex, further reducing its mobility
    \end{answered}
  \item{EMSA says nothing about \textbf{\textit{specific} sequence} bound by a protein, just about a larger fragment. How to \textit{hone in?}}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Directed mutagenesis:} specific subsequence(s) within bound fragment(s) may be targeted for mutation}
      \item{\textbf{Excess oligomer:} relative to the initial fragment(s), smaller hypothesized oligomers for binding can be added, and will \say{soak up} protein binding}
    \end{itemize}
    \end{answered}
  \item{In what 2 main ways may the search space for sequence specificity me reduced?}
    \begin{answered}
    \begin{itemize}
      \item{Prior empirical findings}
      \item{Established sequence conservation}
    \end{itemize}
    \end{answered}
  \item{What are \textit{2 main \textbf{weaknesses}} of EMSA? How may each be addressed?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{\textit{Identity} of binding partner(s):} EMSA doesn't directly provide information about \textit{which specific} protein(s) are binding, just the added size.}
      \item{\textbf{\textit{Identity} of binding partner(s)} may be addressed by \textbf{specific antibody binding;} an antibody will \textit{selectively \textbf{decrease mobility,}} specifically when its complementary protein is bound. This requires \textbf{antibody availability,} though.}
      \item{\textbf{\textit{Sequence specificity:}} EMSA says binding occurs to a fragment, but not \textit{which part(s)} (subsequences/\say{motif(s)}) of the fragment.}
      \item{\textbf{\textit{Sequence specificity}} may be addressed by either \textbf{directed \textit{mutagenesis}} or addition of \textbf{excess putative oligomer}}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{Footprinting Methods}
\begin{QandA}
  \item{Why do footprinting methods \textit{require more robust binding} between DNA probe and protein of interest?}
    \begin{answered}
    Each in this class of methods leverages \textit{\textbf{lack} of signal} as its source of information, so to generate enough \say{background} signal, binding must be robust
    \end{answered}
  \item{What's are \textbf{2 main \textit{weaknesses}} of footprinting methods? How may each be addressed?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Robust requirement:} the requirement for more robust (relative to EMSA) protein binding to probe cannot really be ameliorated.}
      \item{\textbf{Protein identity lack:} as with EMSA, the identity of the protein isn't apparent from footprinting data. Unlike with EMSA, there's not a great way to discern this.}
    \end{itemize}
    \end{answered}
  \item{What's a major strength of footprinting methods relative to EMSA?}
    \begin{answered}
    \textbf{Finer \textit{resolution:}} Each footprinting method provides much more fine-grained detail about \textit{\textbf{where} the protein of interest binds.}
    \end{answered}
  \item{Name and briefly describe each of the \textbf{3 main footprinting methods.}}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{\textit{Nuclease} footprinting:} }
      \item{\textbf{\textit{Chemical} footprinting:} }
      \item{\textbf{\textit{Chemical interference} footprinting:} }
    \end{itemize}
    \end{answered}
  \item{What are the main steps/principles in nuclease and chemical footprinting?}
    \begin{answered}
    \begin{itemize}
      \item{Label DNA with a radioactivity or with a fluorophore}
      \item{Incubation to allow protein binding}
      \item{Introduction of fragment breaks (nuclease or chemical)}
      \item{Run gel elecrophoresis}
      \item{Infer location of protection from \say{hole} in the fragment size distribution (where presumably the protein of interest bound)}
    \end{itemize}
    \end{answered}
  \item{Why should titration be done to aim for about one fragment break per probe?}
    \begin{answered}
    Since the label is at just one end of the DNA molecule, only the length of the first fragment in the linear sequence will be observed.
    \end{answered}
  \item{How does \textbf{chemical \textit{interference}} footprinting work?}
    \begin{answered}
    Chemically induce DNA probe breakoints that distort the molecule and see which do or don't impair binding. Effectful alterations are inferred to be where the protein binds.
    \end{answered}
  \item{The order of which 2 main footprinting steps is reversed for chemical interference footprinting?}
    \begin{answered}
    Induction of fragment breaks comes before incubation for protein binding, since the assay leverages impairment of protein binding by the breaks, rather than the binding's protection from breaks.
    \end{answered}
  \item{Unlike EMSA, with a footprinting method why is there not a straightforward way to tackle protein binding identity?}
    \begin{answered}
    Footprinting leverage signal absence. There's no way to specifically target a void. There's no way to get more information from a null and still have it be a null.
    \end{answered}
\end{QandA}
\section{SELEX}
\begin{QandA}
  \item{What's \textbf{SELEX} stand for?}
    \begin{answered}
    \textbf{SELEX} is the systematic evolution of ligands by exponential enrichment.
    \end{answered}
  \item{Briefly describe the SELEX steps.}
    \begin{answered}
    \begin{itemize}
      \item{1. Create \textit{combinatorial library} of DNA molecules, each with primers flanking a randomized sequence region}
      \item{2. }
    \end{itemize}
    \end{answered}
  \item{Why does SELEX necessarily rely on PCR (amplification)?}
    \begin{answered}
    Each unique molecule in the combinatorial DNA library is very unlikely to bind strongly to the protein of interest, and each molecule that does will be few in number.
    \end{answered}
  \item{Typically, about how large is the combinatorial region of each DNA molecule in the library?}
    \begin{answered}
    Similar to TF motifs (analogous \textit{in vivo} biological application), each combinatorial region is usually about $10-12$ bp. 
    \end{answered}
  \item{What are the main components of each DNA molecule in the combinatorial library?}
    \begin{answered}
    \begin{itemize}
      \item{Each molecule has a $10-12$ bp randomized region in the middle (synthetic binding site to test).}
      \item{Each randomized test region is flanked by primers to facilitate amplification via PCR.}
    \end{itemize}
    \end{answered}
  \item{What are the \textbf{two main methods for purifying/selecting} protein-bound molecules after incubation of DNA with the protein of interest?}
    \begin{answered}
    \begin{itemize}
      \item{Generally, \textbf{EMSA} (electrophoretic mobility shift assay) may be used, as the protein-bound DNA molecules will be larger and more unusually shaped.}
      \item{Alternatively, if an antibody against the protein of interest is available, purification/selection may be done via \textbf{immunoprecipitation.}}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{DNA Replication General}
\begin{QandA}
  \item{\textbf{Replication \textit{safety/security:}} Name and describe the replication machine/enzyme that's tightly cell cycle regulated to assure \textit{exactly one} replication per cycle?}
    \begin{answered}
    \begin{itemize}
      \item{The \textbf{replicative \textit{helicase}} is tightly coupled to the cell cycle to assure loading at necessary locations and only one period of activation per cell cycle.}
      \item{In G1 helicases may be \textit{loaded but not active}; during the remainder of the cell cycle, \textit{loading's prohibited but activation is license.}}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{DNA repair HR}
\begin{QandA}
  \item{What's a major risk of mitotic HR that must be avoided? What danger does it typically pose? Molecularly, how is the solution strategy implemented?}
    \begin{answered}
    \begin{itemize}
      \item{\textit{Loss of heterozygosity (\textbf{LOH})} is a major risk that a mitotic cell must avoid. That's when homologous recombination randomly selects the alternate allele rather than the sister chromatid for recombination.}
      \item{Having 2 copies of each gene safeguards against a defective copy. Losing one copy increases the likelihood of no working copy, which is especially risky for cancer.}
      \item{\textit{\textbf{Cohesin} wraps sister chromatids,} keeping them quite close together. The spatial proximity of sister chromatids relative to homologous chromosomes favors usage of the sister chromatid template.}
    \end{itemize}
    \end{answered}
  \item{If \textit{\textbf{HR} requires a close match} between template and strand to repair, how can \textbf{LOH} occur?}
    \begin{answered}
    Because so much DNA is conserved between individuals, at a small scale with high probability \textit{maternal and paternal alleles will closely match.} The match may be close enough to satisfy HR.
    \end{answered}
\end{QandA}
\section{classic experiments}
\begin{QandA}
  \item{\textbf{\textit{Transforming} principle:} which classical experiment demonstrated that DNA is the \say{transforming principle?} That is, that it's the kind of molecule that transformed bacteria from benign to virulent/pathogenic.}
    \begin{answered}
    The \textbf{Avery-McCarty-MacCleod} experiment showed that DNA is the transforming princple.
    \end{answered}
  \item{Briefly describe the \textbf{Avery-McCarty-MacCleod} experiment and its significance.}
    \begin{answered}
    \begin{itemize}
      \item{1. Smooth bacteria are virulent, rough ones aren't; heat-killed smooth bacteria aren't virulent, nor are heat-killed rough ones.}
      \item{2. Rough bacteria incubated with heat-killed smooth bacteria become virulent.}
      \item{3. When incubating with parts (cell isolates) of heat-killed smooth bacteria, DNA is the only kind of molecule that confers virulence upon the rough bacteria.}
      \item{4. We may infer that \textit{DNA is \say{taken up} by bacteria (as plasmid)} and confers functional properties that other molecules don't.}
      \item{5. This \textit{clarified \textbf{Griffith's 1928 experiment}} that observed the transforming phenomenon when enjecting pairs of rough/smooth dead/live bacteria.}
    \end{itemize}
    \end{answered}
  \item{Which fundamental biomolecular genetics question did the \textbf{Avery-McCarty-MacCleod} experiment address?}
    \begin{answered}
    The \textbf{Avery-McCarty-MacCleod} addressed the \textit{transforming principle,} a sort of \textit{functional heredity,} showing that DNA is the only molecule so empowered.
    \end{answered}
  \item{Which fundamental question did the \textbf{\textit{Meselson-Stahl} experiment} address?}
    \begin{answered}
    The \textbf{{Meselson-Stahl} experiment} addressed the \textit{\textbf{mode} of DNA replication,} demonstrating that it's \textbf{semiconservative.}
    \end{answered}
  \item{\textit{How} did the Meselson-Stahl experiment address the question of mode of DNA replication?}
    \begin{answered}
    \textbf{Isotope labeling:} the experimental design used an alternate \textit{nitrogen isotope}, showing that it's proportional contribution to each DNA molecule diminished over time.
    \end{answered}
  \item{What was the experimental/observational readout from the Meselson-Stahl experiment? That is, how was the proportion of heavier nitrogen assessed?}
    \begin{answered}
    When run on a gel, a DNA molecule with more of the heavier nitrogen isotope moves more slowly and therefore travels a lesser distance.
    \end{answered}
  \item{How did the \textbf{Avery-McCarty-MacCleod} experiment address the alternative hypotheses that protein and/or RNA were responsible for bacterial transformation?}
    \begin{answered}
    \begin{itemize}
      \item{The experiment used various \textit{digestive enzymes} that would break down members of alternative molecular classes like protein or RNA}
      \item{Things like ribonucleases (RNA) and trypsin (protein) that break down some a non-DNA target did nothing to the transformational capability of cellular extract, but so-called deoxyribonucleopolymerase destroyed it.}
    \end{itemize}
    \end{answered}
  \item{Describe the \textbf{Hershey-Chase} experiment.}
    \begin{answered}
    \begin{itemize}
      \item{1. Label both S-35 and P-32, and incubate phages with those labeled isotopes.}
      \item{2. Allow phages to infect cells.}
      \item{3. See which labels go where during pathogenesis/invasion of host cell.}
      \item{4. Sulfur outside $\implies$ protein doesn't really enter.}
      \item{5. Phosphorus inside $\implies$ DNA does enter the cell.}
    \end{itemize}
    \end{answered}
  \item{What was the significance of the \textbf{Hershey-Chase} experiment?}
    \begin{answered}
    The \textbf{Hershey-Chase} experiment further reinforced the idea that DNA is the transforming principle / encoding of heredity.
    \end{answered}
  \item{What biochemical difference between DNA and protein did the \textbf{Hershey-Chase} experiment leverage?}
    \begin{answered}
    DNA contains much more phosphorus than protein, while DNA contains no sulfur (but some amino acids and therefore protein do, plus disulfide bridges). Therefore, labeling phosphorus and sulfur distinguishes protein from DNA.
    \end{answered}
\end{QandA}
\section{connections between topics}
\begin{QandA}
  \item{\textit{\textbf{Immunology} and genome \textbf{integrity/maintenance:}} which process is used by both \textit{V(D)J recombination} and by \textit{DNA repair?}}
    \begin{answered}
    \textbf{NHEJ:} non-homologous end joining is used by both V(D)J recombination and by DNA repair.
    \end{answered}
  \item{\textit{\textbf{Meiosis} and genome \textbf{integrity/maintenance:}} which process is used by both \textit{meiosis} and by \textit{DNA repair?}}
    \begin{answered}
    \textit{Homologous recombination (\textbf{HR})} is used by both meiosis and by DNA repair.
    \end{answered}
  \item{How does the deployment of \textbf{HR} contextually differ?}
    \begin{answered}
    sister chromatids : DNA repair :: homologous chromosomes : meiosis
    \end{answered}
  \item{Motivate the difference in \textit{\textbf{HR} mechanics} that depends on usage context. That is, why do it each way in each context?}
    \begin{answered}
    \begin{itemize}
      \item{For \textbf{DNA \textit{repair,}} a primary concern is \textit{loss of heterozygosity (\textbf{LOH,})} so a chromosome's sister chromatid is favored as the template.}
      \item{For \textbf{meiotic \textit{recombination,}} a primary concern is \textit{generation of genetic \textbf{diversity,}} so the homologous chromosome is favored as recombination partner.}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{DNA Structure}
\begin{QandA}
  \item{What are 4 main functions/traits that DNA must provide?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Replicability} with high \textit{fidelity} and virtually without upper bound on number of replications}
      \item{\textbf{Packaging/Compression:} spatial constraints of the cell and the nucleus means that an otherwise large linear genome must become small.}
      \item{\textbf{\textit{Developmental} regulatory control:} DNA must provide the instructions with which organismal develompent may be orchestrated}
      \item{\textbf{Day-to-day grind:} DNA must provide the instructions that a cell needs to make the proteins that it needs to function each day}
    \end{itemize}
    \end{answered}
  \item{What main problem/question did the double helical structure solve?}
    \begin{answered}
    Replicability/generational passage: the \textit{complementary base pairing} and concomitant redundancy of information addressed the replicability mechanism question.
    \end{answered}
  \item{Which 2 main goals/objectives/purposes of DNA must its packaging balance?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Accessibility:} the DNA must be packed in such a way that access for transcription and replication is easy}
      \item{\textbf{Compression:} the DNA must be packed small enough to fit inside the tiny nucleus.}
    \end{itemize}
    \end{answered}
  \item{Characterize the \textit{bond kinds} involved in the DNA double helix.}
    \begin{answered}
    \begin{itemize}
      \item{In the sugar-phosphate \textbf{backbone} covalent bonds stabilize the double helical structure while the bases bond noncavently, with hydrogen bonds.}
      \item{External : stronger : covalent :: internal : weaker : noncovalent}
      \item{C-G : 3 :: A-T : 2}
    \end{itemize}
    \end{answered}
  \item{Why are A-T bonds \say{weaker} than C-G bonds? What empirical property is often used as a metric for the strength?}
    \begin{answered}
    A-T pair with 2 hydrogen bonds rather than 3. \say{Melting} temperature (Tm) measures this. A-T rich DNA has a lower Tm than G-C rich DNA.
    \end{answered}
  \item{Why are the strands of the DNA double helix said to be \textit{antiparallel?} What gives \textbf{DNA \textit{directionality?}1}}
    \begin{answered}
    \textit{Asymmetry} in the chemical bonding linking together deoxybiose subunits of the \textit{backbone} is what gives DNA directionality and the \say{antiparallel} moniker.
    \end{answered}
  \item{What are some benefits of having antiparallel strands?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{\textit{Proofreading} mechanism:} the 5'-to-3' \textit{exnucleolytic proofreading} that allows DNA polymerase to boost its fidelity is enabled by DNA's directionality.}
      \item{\textbf{Coding space:} while most relevant for prokaryotes and viruses with simple genomes, DNA's directionality means that \textit{genes may overlap}}
    \end{itemize}
    \end{answered}
  \item{About what percentage of cell volume is the nucleus?}
    \begin{answered}
    About 10 percent.
    \end{answered}
  \item{What's a major logistical benefit of having a nucleus and nuclear envelope (i.e., walling off DNA from the rest of the cell)?}
    \begin{answered}
    The nucleus is an instance of the more general strategy of \textit{compartmentalization,} making desirable \textit{molecular collisions more likely.} It shifts the \textit{probability distribution}  of molecular collisions in a productive way.
    \end{answered}
  \item{Characterize the percentage of the human genome devoted to protein coding vs. repetitive elements}
    \begin{answered}
    \begin{itemize}
      \item{Protein coding genome sequence is just about 1.5 percent of the total.}
      \item{Repetitive, transposable elements account for nearly half of the human genome.}
    \end{itemize}
    \end{answered}
  \item{How large is the average human gene, in total nucleotides base pairs and in amino acids (how main coding nucleotides?)}
    \begin{answered}
    \begin{itemize}
      \item{The average human gene spans abut 27 Kb.}
      \item{The average protein is only about 430 amino acids long, so about 1300 coding nucleotides.}
    \end{itemize}
    \end{answered}
  \item{Briefly, what are the 3 main specialized regions of each chromosome that regulate its structure?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Replication origins:} many in eukaryotes to expedite replication of huge genomes; typically (always?) just one in prokaryotes}
      \item{\textbf{Centromere:} critically, there's \textit{one and \textbf{only one.}} Mitotic segregation won't occur if there's no centromere, chromosomes will fail to properly segregate during mitosis.}
      \item{\textbf{Telomere:} distiguishing chromosome ends from breaks so that normal, healthy DNA tips aren't mistaken for DNA damage and then erroneously repaired}
    \end{itemize}
    \end{answered}
  \item{DNA replication creates a structural problem for chromosomes: how to know when to stop and how to finish? How do prokaryotic and eukaryotic cells differ in their solution strategies?}
    \begin{answered}
    \begin{itemize}
      \item{Prokaryotes use a \textit{circular genome.}}
      \item{Eukaryotes use linear chromosomes and specialized end structures (telomeres).}
    \end{itemize}
    \end{answered}
  \item{Describe the central structure of a nucleosome?}
    \begin{answered}
    The nucleosome core is a \textbf{histone \textit{octamer,}} composed of a H3-H4 tetramer and a pair of H2A-H2B dimers.
    \end{answered}
  \item{Approximately what fold compression does packing into nucleosomes provide for DNA packing?}
    \begin{answered}
    Wrapping around nucleosomes 
    \end{answered}
  \item{About how long is linker DNA?}
    \begin{answered}
    Linker DNA ranges from just a few base pairs to up to about eight base pairs, so the distance between nucleosomes is about $200$ bp.
    \end{answered}
  \item{Describe size and conservation of histones relative to other proteins.}
    \begin{answered}
    Histones are smaller and more conserved than most other proteins. They're only about $102-135$ amino acids, and differ by only a few amino acids, suggesting critical structural importance under intense stabilizing selection
    \end{answered}
  \item{What features do the histone proteins share?}
    \begin{answered}
    Histones share the \textit{histone \textbf{fold,}} a domain composed of three $\alpha$ helices connected by short loops
    \end{answered}
  \item{Account for the simultaneous strength and nonspecificity of affinity between DNA and the histone octamer. Why does virtually any sequence bind so strongly?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Charge} and \textit{amino acid composition:} DNA is negatively charged, while about $1/5$ of the histone's amino acids are either \textit{lysine} or \textit{arginine.}}
      \item{Basic amino acids are positively charged at neutral pH (because they donate hydroxide ions to solution), so they modest positive charge of the histone core attracts negatively charged DNA.}
    \end{itemize}
    \end{answered}
  \item{Describe the extent of the interface between the histone octamer and the DNA that wraps around it.}
    \begin{answered}
    \begin{itemize}
      \item{In just $147$ bp, or about one-and-a-half \say{wraps} around the octamer, DNA makes a whopping $142$ hydrogen bonds with the octamer.}
      \item{Roughly half of the hydrogen bonds between DNA and the octamer are between the backbone and the histone core (not tails).}
    \end{itemize}
    \end{answered}
  \item{Characterize the nucleosome's sequence binding preference.}
    \begin{answered}
    Certain dinucleotide pairs bend with relative ease; the DNA is more sharply bent on the nucleosome-adjacent side, so on that side, it's preferable to have A-T enrichment in the octamer-proximal minor groove.
    \end{answered}
  \item{Why must the sequence binding preference of nucleosomes be relatively weak?}
    \begin{answered}
    Nucleosomal sequence binding preference may be viewed as an assist with the chromatin architectural plan, but greater functional exigency must be given preference for nucleosome positioning and DNA access, so nucleosome preference must not overwhelm TFs' ability to reposition them.
    \end{answered}
\end{QandA}
\section{DNA Replication}
\begin{QandA}
  \item{What's it mean that \textit{DNA replication is \textbf{semiconservative?}}}
    \begin{answered}
    \textit{\textbf{Semiconservative} replication} means that in each \say{daughter duplex} that's generated, one strand is newly synthesized, and one strand is inherited from the parent cell.
    \end{answered}
  \item{Which \textit{classic experiment} demonstrated that \textit{DNA replication is \textbf{semiconservative?}}}
    \begin{answered}
    The \textbf{Meselson-Stahl experiment} demonstrated the DNA replication is semiconservative.
    \end{answered}
\end{QandA}
\section{Chromatin Histones Nucleosomes}
\begin{QandA}
  \item{If DNA so frequently \say{breathes} by unwrapping briefly from the nucleosome (about 4 times per second, with each exposure $10-50$ms), why do eukaryotic cells need ATP-dependent chromatin remodeling complexes at all?}
    \begin{answered}
    While contact between a TF and the DNA may be able to happen randomly in such frequent but brief windows, likelihood improves greatly with the remodeling factors, as they maintain accessibility much longer.
    \end{answered}
  \item{How do nucleosome remodeling complexes achieve \textit{nucleosome replacement?} What's the approximate average frequency of that turnover in cells?}
    \begin{answered}
    \begin{itemize}
      \item{Remodeling complexes feature a \textit{\textbf{helicase-like} subunit} that uses \textbf{ATP hydrolysis} to achieve the \say{sliding} action.}
      \item{Remodeling complexes associated/collaborate with \textit{histone \textbf{chaperone} proteins} that assist with histone substitution of histones within the octamer.}
      \item{On average, each nucleosome is replaced once every $1-2$ hours, or around $15$ times each day.}
    \end{itemize}
    \end{answered}
  \item{What's the most important factor for determining nucleosome position? Why is this functionally important?}
    \begin{answered}
    \begin{itemize}
      \item{Binding of transcription factors and other proteins, much more than histone-to-sequence binding preference, influences nucleosome position}
      \item{For functional plasticity and response to signaling / changing conditions, it's critical that DNA-binding proteins' influence outweighs the (static) preference of histones for certain DNA sequence features.}
    \end{itemize}
    \end{answered}
  \item{Which histone has been less conserved through evolutionary history? How may this help explain interspecific chromatin structural differences?}
    \begin{answered}
    The \textbf{\say{linker} histone (H1)} has been much less conserved than the others. Since it plays a more prominent role in higher-order chromatin structure, that lack of conservation may help account for interspecific differences in chromatin packing.
    \end{answered}
  \item{How does linker histone H1 facilitate chromatin condensation into higher-order nucleosomal arrays?}
    \begin{answered}
    H1 \textit{alters DNA exit path} from the nucleosome, in a way making the emergent DNA slightly more tethered/less flexible.
    \end{answered}
  \item{It's said that \textit{heterochromatin is self-propagating.} What early experimental observation in flies supports this?}
    \begin{answered}
    \begin{itemize}
      \item{The so-called \textit{position effect} from early fly genomics experiments supports the notion that heterochromatic state propagates}
      \item{Euchromatin translocated into a heterochromatic neighborhood has its genes \textit{silenced}.}
    \end{itemize}
    \end{answered}
  \item{\textbf{Histone PTMs and accessibility:} which general histone tail modification is associated with chromatin accessibility? How does it foster access?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Lysine \textit{acetylation}} is strongly associated with chromatin accessibility, by \textit{charge neutralization.}}
      \item{Specifically, the negative acetyl charge neutralizes the positive lysine charge, thereby weakening binding affinity between DNA and the the octamer.}
    \end{itemize}
    \end{answered}
  \item{\textbf{Histone PTMs and chromatin:} Which histone PTM is strongly associated with condensed heterochromatin? How so?}
    \begin{answered}
    H3K9me3 is most associated with (esp. constitutive) heterochromatin. \textbf{Linker histone H1} is recruited by this PTM.
    \end{answered}
  \item{How do cells unevenly distribute histone protein synthesis? How does this relate to fulfillment of functional needs?}
    \begin{answered}
    \begin{itemize}
      \item{\textit{Histone variants} (most) are synthesized continually \textit{throughout interphase.}}
      \item{\textit{Canonical histones} are synthesized primarily during \textit{S phase.}}
      \item{Histone variants, by nature, typically replace a canonical histone to shift probability distributions of events in some functional way; as such, they're needed throughout much of the cell cycle.}
      \item{Synthesizing canonical histones primarily during S phase is both functionally efficient and tips the concentration of available histone protein for nucleosomes toward the canonical/constitutive rather than the specifically functionally facultative.}
    \end{itemize}
    \end{answered}
  \item{Describe how nucleosome remodeling complexes attain functional two kinds of specificity.}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{\textit{Spatial} specificity:} nucleosome remodeling complexes target specific genomic regions / motifs by incorporating \textit{DNA-binding subunit(s).}}
      \item{\textbf{\textit{Functional} specificity:} remodeling complexes favor binding to a particular histone variant through \textit{protein-protein interactions;} specifically, complexes include subunit(s) for interaction with chaperone(s) associated with particular histone(s).}
    \end{itemize}
    \end{answered}
  \item{Broadly, what balance does a structure featuring a dense core but several looser, freer tails (histones) strike, and functionally what does this achieve?}
    \begin{answered}
    \begin{itemize}
      \item{The balance/tradeoff is struck between density and close spatial packing, and dynamism.}
      \item{Pairing a small, dense core with several short but flexible \say{tails} allows histones to both condense DNA into chromatin while leaving a \say{hook,} or \say{platform} for functional alteration as cellular needs change.}
    \end{itemize}
    \end{answered}
  \item{\textbf{Histone PTM \textit{combinatorial specificity:}} How do nucleosome remodeling complexes and other \say{readers} of chromatin implement combinatorial specificity?}
    \begin{answered}
    Many/all individual PTMs have at least one \say{reader} motif/domain. Complexes or large proteins can achieve a sort of combinatorial mark specificity and stronger binding by \textit{incorporating several domains/modules} with \say{reading} properties.
    \end{answered}
  \item{\textbf{Chromatin \textit{spreading:}} what's the classic way in protein/enzymatic complexes \say{spread} chromatin marks?}
    \begin{answered}
    A complex that \textit{pairs reader with writer} is effective for \say{spreading} a particular mark. That is, a complex in which the writer leaves a mark that the reader binds.
    \end{answered}
\end{QandA}
\section{math stat general}
\begin{QandA}
  \item{What's a good way to measure how quickly an object converges to 0 in some limit? That's is, what's a general strategy? Think \textbf{Blitzstein} and \textbf{Mark Low.}}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{\textit{Competition} between terms:} specifically, multiply by a \textit{sufficiently large power} of $n$ going to infinity.}
      \item{More generally, allow a term going to infinity to \say{compete with} a term going to 0, until the nonzero term is sufficiently powerful for nondegenracy.}
    \end{itemize}
    \end{answered}
  \item{How and under what conditions can we \textit{define a \textbf{natural ordering of matrices?}}}
    \begin{answered}
    \begin{itemize}
      \item{A natural ordering for matrices occurs when they're all squares of a common size and are positive definite}
      \item{The determinant (product of eigenvalues) then defines a natural ordering of the matrices.}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{Virology2020 L07 Transcription and RNA processing}
\begin{QandA}
  \item{Which \textbf{molecule type} (of nucleic acid) is required for transcription?}
    \begin{answered}
    \textit{Transcription} is a process linked by definition to \textbf{dsDNA}; dsDNA to mRNA.
    \end{answered}
  \item{Which viral classes involve transcription at all?}
    \begin{answered}
    The classes that \textit{include \textbf{dsDNA}} as part of the lifecycle use transcription; Class I, II, VI, and VII.
    \end{answered}
  \item{Which viruses enter the cell \say{ready to go} w.r.t. transcription? That is, which viruses \textit{don't} need intermediate(s)?}
    \begin{answered}
    \textbf{Class I, dsDNA}: The dsDNA viruses enter the cell ready for transcription, since their genome is the molecular input for transcription.
    \end{answered}
  \item{Which viruses are even more \say{ready-to-go} from the protein production perspective than the \say{ready-to-go} dsDNA viruses? That is, which viruse \say{skip} straight to translation?}
    \begin{answered}
    \textbf{Class IV, sense-strand RNA} viruses enter the cell even more \say{ready} for protein production, as they have the property that \textit{gRNA = mRNA}.
    \end{answered}
  \item{What are the 2 main types of viral \textbf{transcription regulatory frameworks}? Describe each.}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{\textit{Cascade} regulation}: Genes may have a \textit{serial} expression structure, in space and/or in time. One or more \say{upstream} gene products activates/licenses a \say{downstream} gene.}
      \item{\textbf{\textit{Feedback} regulation}: One gene may \textit{\textbf{auto}regulate}; this could be negative or positive; more about \textit{how much} than abotu ON/OFF.}
    \end{itemize}
    \end{answered}
  \item{What are some examples of \textbf{modular organization} (in \textit{function} and in \textit{sequence} of sequence-specific transcriptional activators?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{DNA binding}: most N-terminal region of the protein; often a \textit{zinc finger}, a \textit{helix-turn-helix} motif, or an especially \textit{basic (ph)} region}
      \item{\textbf{Dimer formation}: often implemented via \textit{leucine zipper}}
      \item{\textbf{NLS}: nuclear localization sequence, used to direct a transcription factor back to the nucleus to do its job}
      \item{\textbf{Activation}: most C-terminal region; often a \textit{acidic} or rich in glutamine, proline, or isoleucine}
    \end{itemize}
    \end{answered}
  \item{What are 3 common \textbf{\textit{molecular} implementations} of the \textbf{DNA binding} portion of a transcriptional activator?}
    \begin{answered}
    \begin{itemize}
      \item{\textit{Zinc finger}: }
      \item{\textit{Helix-turn-helix}: a particular structural motif}
      \item{\textit{Basic}: a region enriched for basic amino acids}
    \end{itemize}
    \end{answered}
  \item{What's the primary purpose of early vs. late phases? That is, why have phases?}
    \begin{answered}
    Partitioning the life cycle compartmentalizes gene expression according to life cycle timing, avoids the otherwise wasteful output, and economizing the life cycle.
    \end{answered}
  \item{What kind/class of protein is particularly common in the early phase?}
    \begin{answered}
    Transcriptional activators and regulatory proteins are especially likely to be in the early phase.
    \end{answered}
  \item{What kind of proteins are especially associated with the late phase? Why?}
    \begin{answered}
    Structural proteins are typically late-phase, since they're only needed for particle assembly when it's time to infect other cells.
    \end{answered}
  \item{What viral \textbf{life cycle \textit{molecular process}} is especially associated with coupling to the activation of \textit{	extbf{late-phase} transcription}?}
    \begin{answered}
    A common viral strategy is to \textit{couple \textbf{late}-phase transcription to \textbf{genome replication}}, as replication signals that it's about time to \textit{form new particles} and infect other cells.
    \end{answered}
  \item{\textbf{SV40}: Descibe the general transcriptional regulatory strategy of SV40.}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{SV40} has a \textit{circular genome} with just a \textit{\textbf{single, bidirectional} promoter} that drives transcription.}
      \item{\textbf{SV40} \textit{physically bifurcates transcriptional phases}; that is, the early- and late-phase transcriptionl programs are implemented by \textit{procession in \textbf{opposite directions}} from the single promoter.}
      \item{\textbf{SV40} may use a \textit{micro-RNA}-based strategy to suppress expression of early-phase transcripts during the late phase, with the late-phase transcriptional direction encoding a miRNA that seems to antagonize early-phase gene translation.}
    \end{itemize}
    \end{answered}
  \item{Which host transcriptional activator does adenovirus hijack to drive transcription? How does it do it? What can that cause for the host?}
    \begin{answered}
    \begin{itemize}
      \item{Adenovirus \textbf{E1A} protein \textit{boots \textbf{retinoblastoma (Rb)}} protein from \textbf{E2f}, which adenovirus uses to drive transcription.}
      \item{\textit{\textbf{Rb} recruits HDACs}, so the eviction of Rb from a locus an allow histone acetylation and gene activation, which can promote \textit{cell division and \textbf{cancer}}.}
    \end{itemize}
    \end{answered}
  \item{SV40 and adenovirus share in common the regulatory trait that \textit{\textbf{DNA synthesis} helps stimulate transition to \textbf{late-phase} transcription}. Desribe the main difference between SV40 and adenovirus in late-phase transition.}
    \begin{answered}
    Adenovirus uses two proteins (\textbf{IVa2} and \textbf{L4}) as positive regulators of late-phase transcription, rather than just the DNA synthesis based stimulation of transition to late-phase transcription.
    \end{answered}
  \item{What's a distinguishing feature of herpesvirus infection, relative to SV40 and to adenovirus, and with respect to \say{getting started} by initiating the life cycle and transciptional program?}
    \begin{answered}
    \textbf{Pre-prepared protein}: Unlike SV40 and adenovirus, which must make \textit{make} a protein to get started, herpesvirus comes packaged with the \textbf{VP16} protein pre-made, between the capsid and the envelope.
    \end{answered}
  \item{Why is \textit{escape from the nucleus} so vitally important in the life of many mRNA molecules?}
    \begin{answered}
    For much of the viral life cycle, mRNA needs to be \textit{translated}, and the \textit{ribosomes are in the cytosol}. This means that the mRNA molecules must be exported from the nucleus for translation.
    \end{answered}
  \item{How do RNA molecules normally make it out of the nucleus and into the cytosol?}
    \begin{answered}
    RNA, like proteins and other macromolecules, is \textit{too large to passively translocate} from nucleus to cytosol; instead, RNA molecules often have \textit{nuclear export signals} that are then \textit{bound by \textbf{exportin}} proteins medaite passage by \textit{interactive with the \textbf{nuclear pore complex (NPC)}} proteins.
    \end{answered}
  \item{Why must some viral RNA make it out of the nucleus unspliced?}
    \begin{answered}
    If a virus has a RNA genome and assembles into particles in the cytosol, then the full-length RNA genome must get out into the cytosol for packaging into new virus particles.
    \end{answered}
  \item{How does the discovery of \textbf{alternative splicing} relate to virology?}
    \begin{answered}
    \begin{itemize}
      \item{Alternative splicing was discovered in \textbf{adenovirus}, as researchers realized that mature (spliced), cytosolic RNA tended to be smaller than many of the younger RNA molecules in the nucleus.}
      \item{Upon closer examination with the electron microscope, it was discovered that adenovirus mRNA-DNA duplexes had large DNA loops not hybridized to the RNA; these were introns that had been spliced out.}
    \end{itemize}
    \end{answered}
  \item{How does SV40 leverage a host cell protein to regulate part of the timing of its lifecycle / transcriptional program?}
    \begin{answered}
    SV40 uses a host cell protein as a repressor that binds late-phase genes.
    \end{answered}
  \item{How does SV40 \textit{couple transcription regulation to genome replication}?}
    \begin{answered}
    \begin{itemize}
      \item{SV40 uses a \textbf{dilution}-like mechanism to couple DNA synthesis (genome replication) to transcription program timing.}
      \item{Specifically, the \textit{level of the host cell protein remains constant}. Once SV40 genome replication begins, though, the ration of repressor to genome copies declines, and there's an increasing availability of DNA molecules that lack repressive binding of the host cell protein to late-phase genes.}
    \end{itemize}
    \end{answered}
  \item{For viruses, what's a major advantage of splicing?}
    \begin{answered}
    As with host cells, \textit{splicing generates \textbf{transcriptome diversity}}. The \textbf{genome economy} this facilitates, allowing the virus a compact but information-rich genome, is valuable.
    \end{answered}
  \item{What's the general order of operation of the main proteins involved in miRNA processing?}
    \begin{answered}
    \begin{itemize}
      \item{1. \textbf{DROSHA} forms the initial, \textit{primary micro-RNA (pri-miRNA)}.}
      \item{2. \textbf{DICER} processes pri-miRNA to produce mature miRNA.}
      \item{3. \textbf{Argonaute} binds miRNA and is related to effector function.}
    \end{itemize}
    \end{answered}
  \item{How are \textbf{\textit{circular} RNA} molecules generated (by what molecular process)?}
    \begin{answered}
    \textit{Backsplicing} generates circular RNA molecules, by joining the 3' end of a downstream exon to the 5' end of an upstream exon.
    \end{answered}
\end{QandA}
\section{Virology2020 L09 Reverse transcription and integration}
\begin{QandA}
  \item{Affirm or refute: any virus that contains reverse transcriptase (RT) is a retrovirus.}
    \begin{answered}
    \textit{False}: \textit{Retroviridae} is Baltimore Class VI, but Baltimore Class VII also has RT.
    \end{answered}
  \item{Affirm or refule: any virus that contains reverse transcriptase (RT) has a \textit{RNA} genome.}
    \begin{answered}
    \begin{itemize}
      \item{\textit{False}: Baltimore Class VII viruses have RT but use a dsDNA genome (dsDNA-RT viruses)}
      \item{The dsDNA-RT (Class VII) families are \textit{Hepadnaviridae} and \textit{Caulimoviridae}.}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{random virology}
\begin{QandA}
  \item{Give at leas two different solutions to the problem of \textit{targeting a \textbf{protein} to a \textbf{membrane.}}}
    \begin{answered}
    \begin{itemize}
      \item{Retroviruses use \textbf{myristate,} a lipid / fatty acid compound, conjugated onto the 5' end of a protein after translation}
      \item{Some viruses (and cell proteins) us a \textbf{signal sequence} of amino acids within the protein itself.}
    \end{itemize}
    \end{answered}
  \item{\textbf{Koch's postulates}: aims -- what's the main \textit{purpose} of considering and investigating Koch's postulates?}
    \begin{answered}
    
    \end{answered}
  \item{\textbf{Koch's postulates}: what are Koch's (4) postulates?}
    \begin{answered}
    \begin{itemize}
      \item{1. \textit{Sensitivity and specificity}: The microbe must be found in affected individuals and not in others.}
      \item{2. \textit{Isolation and culture}: The microbe must be able to be isolated from affected individuals and grown in lab culture.}
      \item{3. \textit{New infection}: The harvested, cultured microbe must be able to infect a new host into which it's experimentally introduced.}
      \item{4. \textit{Re-harvest and identify}: The microbe must be able to be isolated from the experimentally infected individual, and shown identical to the originally isolated microbe.}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{TWiV96 Making Viral DNA}
\begin{QandA}
  \item{\textbf{Main steps}: What are the 4 main steps of viral DNA replication?}
    \begin{answered}
    \begin{itemize}
      \item{1. \textit{Recognition}: DNA replication is \textit{spatially nonrandom}; that is, it begins at one or more defined places within the virus genome.}
      \item{2. \textit{Initiation}: (almost) all DNA synthesis requires a \textit{primer}, and the mechanics/logistics of initiation vary largely w.r.t. what priming strategy a virus uses.}
      \item{3. \textit{Elongation}: Once replication has been initiated, polymerase(s) elongate(s) DNA; in the case of a replication fork, there's typically quite different enzymatic machinery for each of the strands (leading and lagging)., }
      \item{4. \textit{Termination}: Typically (?) by simply running out of template, replication eventually ceases.}
    \end{itemize}
    \end{answered}
  \item{\textbf{Priming}: What are the 3 main strategies for \textit{priming} DNA synthesis? How does this fundamentally differ from RNA viruses?}
    \begin{answered}
    \begin{itemize}
      \item{\textit{\textbf{RNA} priming}: Some(Specialized \textit{DNA-dependendent RNA polymerases} called \textbf{primases} can synthesize a short RNA oligomer complementary to a DNA template, providing the essential free 3'-OH for DNA polymerase to prime off of.)}
      \item{\textit{\textbf{DNA} priming}: Some(Some viruses build the free 3'-OH for DNA polymerase right into their genome.)}
      \item{\textit{\textbf{Protein} priming}: Some(Some viruses have a DNA-binding protein that coordinates with DNA polymerase and possible other proteins to ignite DNA synthesis.)}
      \item{RNA synthesis often doesn't required a primer, so this is a less essential aspect of consideration for RNA viral study.}
    \end{itemize}
    \end{answered}
  \item{Provide an example of each of the three main priming strategies.}
    \begin{answered}
    \begin{itemize}
      \item{\textit{\textbf{RNA} priming}: Some(Eukaryotic, linear chromosomes use RNA priming (\textbf{primase}, a specialize type of \textit{DNA-dependent RNA polymerase}, or DdRp))}
      \item{\textit{\textbf{DNA} priming}: Some(\textit{Parvoviridae} members like AAV have an exposed 3'-OH at the tail of each inverted terminal repeat (ITR) hairpin end, so the primer's built right into the genome.)}
      \item{\textit{\textbf{Protein} priming}: Some(\textit{Adenovirus} uses protein priming, with \textbf{preterminal protein (pTP)} working in conjunction with polymerase.)}
    \end{itemize}
    \end{answered}
  \item{\textbf{\textit{Elongation} examples}: Provide example viruses for each of the two main elongation stratgies.}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Replication fork}: Some(SV40)}
      \item{\textbf{Strand displacement}: Some(\textit{Parvoviridae} and \textit{Adenoviridae})}
    \end{itemize}
    \end{answered}
  \item{\textit{Alternative \textit{elongation} strategy}: What's a major alternative strategy used for elongation during RNA replication, in a way \textit{combining the two major strategies}?}
    \begin{answered}
    \say{Rolling circle} replication is a strategy used by many viruses for DNA genome replication.
    \end{answered}
  \item{\textbf{\textit{Rolling circle} replication}: provide an example of a virus/es that use \textit{rolling circle} replication.}
    \begin{answered}
    \begin{itemize}
      \item{Human herpesvirus 6 (HHV-6)}
      \item{Human papillomavirus-16 (HPV-16)}
    \end{itemize}
    \end{answered}
  \item{\textbf{Universal truths}: What are several things about viral DNA replication that always (or nearly always) hold true?}
    \begin{answered}
    \begin{itemize}
      \item{\textit{Priming}: DNA synthesis (nearly) always requires a primer.}
      \item{\textit{Directionality}: DNA synthesis always occurs in the 5'-to-3' direction, reading the template 3'-to-5'.}
      \item{\textit{At least one protein}: All viruses need the host to make at least one protein for them. The virus can encode a DNA polymerase, and if it doesn't it needs a protein to \say{orchestrate the host} to get viral DNA made rather than the excess of host DNA.}
    \end{itemize}
    \end{answered}
  \item{\textbf{Fundamental problem}: What fundamental problem arises for \textit{linear DNA} genome replication? Why for \textit{DNA}?}
    \begin{answered}
    \begin{itemize}
      \item{\textit{The 5' end problem} refers to the difficulty of copying the DNA at the 5'-most end of the template.}
      \item{The 5' end problem arises specifically in \textit{DNA} viruses because of the \textit{priming requirement}. RNA viruses don't in general face the same challenge.}
    \end{itemize}
    \end{answered}
  \item{For humans (and mostly all other linear genome eukaryotes?) what's the solution to the 5' end problem?}
    \begin{answered}
    Telomeres solve the 5' end problem by signaling the end of chromosomes, with the support of specific sequence maintenance and DNA-binding enzymes and other proteins.
    \end{answered}
  \item{\textbf{Bare (protein) necessities}: what kinds of proteins provide the minimal activities needed for viral DNA genome replication?}
    \begin{answered}
    \begin{itemize}
      \item{\textit{Recognition}: the virus needs at least one protein to recognize its ori(s), so \textbf{origin-binding protein(s)}.}
      \item{\textit{Unwinding}: the virus needs \textbf{helicase(s)} to unwind the DNA.}
      \item{\textit{Elongation}: the virus needs \textbf{DNA polymerase(s)} for the main work of DNA synthesis.}
      \item{\textit{Chew back}: The virus also needs \textbf{exonucleases} to chew back DNA.}
      \item{\textit{Metabolism}: The virus depends on various \textbf{nucleic acid metabolic enzymes}.}
    \end{itemize}
    \end{answered}
  \item{What's the general relationship between genome size and independence from, vs. reliance on, the host cell?}
    \begin{answered}
    In general, \textit{smaller viruses \say{orchestrate}} while \textit{larger viruses \say{encode}}; that is, a smaller genome encodes protein(s) that manipulate host machinery while a larger genome encodes machinery specifically for the virus.
    \end{answered}
\end{QandA}
\section{Molecular Popgen}
\begin{QandA}
  \item{What are the two main \textbf{axes/forms of violation of the Hardy-Weinberg law's conditions?} Equivalently, what are the two main kinds of conditions?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Internal} to the population: endogenous forces like \textit{nonrandom mating} and various molecular mechanistic biases.}
      \item{\textbf{External} to the population: exogenous forces include \textit{gene flow}, e.g. from \textit{migration} or \textit{horizontal gene transfer}, or \textit{natural selection.}}
    \end{itemize}
    \end{answered}
  \item{What are some broad kinds of \textbf{molecular mechanistic biases} that cause violation of Hardy-Weinberg conditions?}
    \begin{answered}
    \begin{itemize}
      \item{\textit{Gametogenesis}: certain alleles may skew the distribution of alleles by meiotic drive, whether by \textit{covariance} or by \textit{disproportionality}.}
      \item{\textit{Fertilization}: certain alleles may give a particular gamete an unusually high or low probability of winning the Fertilization race.}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{Hardy Weinberg}
\begin{QandA}
  \item{What are the \textbf{Hardy-Weinberg} law's three \textit{fundamental consequences/conclusions,} that hold when conditions are met?}
    \begin{answered}
    \begin{itemize}
      \item{1. Allele frequencies never change}
      \item{2. Genotype frequencies change, at most, only in the first mating, remaining constant thereafter.}
      \item{3. Allele frequencies and genotype frequencies are mutually/bidirectinoally determined, so that knowledge of value(s) in one domain directly maps to knowledge of certain value(s) in the other domain.}
    \end{itemize}
    \end{answered}
  \item{What's the general mathy strategy for proving the Hardy-Weinberg law?}
    \begin{answered}
    The Hardy-Weinberg is logically proved by, for each progeny/offspring case, taking a \textit{weighted average of conditional probabilities}, with offspring genotype conditions on parental mating genotypes, weighted by the mating's probability.
    \end{answered}
  \item{What's a useful frame in which to prove the Hardy-Weinberg law?}
    \begin{answered}
    It's useful to frame the proof by considering and classifying cases by \textit{\textbf{number} and \textbf{kind} of heterozygotes} in each mating.
    \end{answered}
  \item{\textbf{Assumptions and justification}: what assumption(s) justify the computation of \textit{genotype frequency} from allele frequency (i.e., a simple product)?}
    \begin{answered}
    \textbf{Conjectured \textit{independence}}: it's the assumed independence, both during \textit{gametogenesis} and during \textit{fertilization}, that justifies this. Specifically, alleles are equally likely to make it into gametes, and gametes of different haplotypes are equally likely to win the \say{ferilization race.}
    \end{answered}
  \item{\textbf{Assumptions and justification}: what assumption(s) about the population dynamics, structure, etc. justify the doubling the \textit{genotype probability} to map from ordered to unordered (e.g., $f(A_i A_j) = 2P_{ij}$)?}
    \begin{answered}
    \textbf{\textit{Monoecious}, or \textit{balanced allele source frequencies}}: If we regard the two \say{slots} of a diploid genotype as maternal and paternal alleles, then doubling the ordered probability is justified by the assumed equiprobability of a particular allele coming from either parent.
    \end{answered}
  \item{\textbf{Assumptions and justification}: what assumption(s) justify the use of symmetry in computation of \textit{mating probavility} (multiplying by 2))?}
    \begin{answered}
    \textbf{\textit{Monoecious}, or \textit{balanced genotype frequencies}}: \say{reflecting} the genotypes in the mating and as such doubling the mating probability to go from orered to unordered reflects the assumption of either a monoecious population (no distinction between parents), or \textit{equal genotype distributions} between the population's sexes.
    \end{answered}
\end{QandA}
\section{HIV Nature 2020 RNA structure DREEM}
\begin{QandA}
  \item{Which important part of the HIV genome seems to exist in a roughly 2:1 or 3:1 ratio of 4- and 5-stem structures?}
    \begin{answered}
    The \textbf{Rev response element (RRE)} exists mainly in this ratio of stem counts.
    \end{answered}
  \item{What two tools were combined to conduct this study?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{DMS-MaPseq} for the biological, experimental work (methyl adducts to the exposed A and C bases, then random mutations from the RT)}
      \item{\textbf{DREEM} algorithm for inferring structure from the random mutation data coming out of DMS-MaPseq}
    \end{itemize}
    \end{answered}
  \item{What are the two main challeges with the use of DMS-MaPseq, related to the biochemistry of the experiment?}
    \begin{answered}
    \begin{itemize}
      \item{\textit{Random, low-frequency mutation}: the mutations from the TGIRT-III reverse transcriptase are random and at only $2-10\%$ of methylated sites.}
      \item{\textit{Sequence context dependence}: the frequency of the RT's mutations that provide the accessibility signal depends on the sequence context.}
    \end{itemize}
    \end{answered}
  \item{What does this study show about the relationship between biological context (in vitro, in virion, and in infected cells) about RRE structure?}
    \begin{answered}
    Proportions of the two main RRE structural variants were consisitent across biological contexts, suggesting that the structure is relatively invariant w.r.t. context.
    \end{answered}
  \item{Which site in the HIV genome was used to study alternative splicing?}
    \begin{answered}
    The \textbf{A3 splice site}'s structure was studied to assess the structural impact on alternative splicing?
    \end{answered}
  \item{What are the main findings regarding structure's effect on alternative splicing?}
    \begin{answered}
    \begin{itemize}
      \item{A3 splice site exists in two main conformations, one with the splice site exposed and one with it hidden in a base-paired structure.}
      \item{Other sequences near the A3 splice site (e.g., splice enhancers) also strongly influence splice site use by their structural conformation.}
      \item{A3 splice site usage is heavily influenced by structure, with the self-paired structure much less amenable to splicing.}
    \end{itemize}
    \end{answered}
  \item{How was structure manipulated to investigate alternative splicing?}
    \begin{answered}
    Mutations were introduced that stabilized the self-base-paired loop structure that hides the A3 splice site.
    \end{answered}
  \item{Which HIV protein's mRNA level is entirely regulated by the A3 splice site and therefore very affected by structure there?}
    \begin{answered}
    The \textit{Tat} protein's mRNA only arises when the A3 splice site is used.
    \end{answered}
\end{QandA}
\section{HIV Nature 2010 thymic selection and protective MHC allele}
\begin{QandA}
  \item{What three immunodevelopmental processes/phenomena are crucial for understanding this paper?}
    \begin{answered}
    \begin{itemize}
      \item{MHC restriction}
      \item{\textit{Positive} selection of T cells during development in thymus}
      \item{\textit{Negative} selection of T cells during development in thymus}
    \end{itemize}
    \end{answered}
  \item{What's the name/phrase for the HIV patients who keep virus low even without anti-retroviral therapy (i.e., the human subjects of this study)?}
    \begin{answered}
    \say{Elite controllers} are HIV patients capable of suppressing viral load even without ART assistance.
    \end{answered}
  \item{What's the name of the protective MHC allele?}
    \begin{answered}
    HLA-B57
    \end{answered}
  \item{Why is the protective MHC allele protective?}
    \begin{answered}
    \begin{itemize}
      \item{This allele binds fewer self-peptides than most, implying that it presents a less diverse}
      \item{Presentation of fewer self-peptides during thymic selection by this allele leads to a batch of cognate TCRs enriched for few, but strong, peptide-contacting residues}
      \item{A TCR pool reliant on relatively few (1-3) individual contacts with a peptide is inherently more \textit{robust to mutations,} as a given amino acid substitution is less likely to lower the interaction between the TCR and the peptide:MHC complex enough to ablate recognition by the T cell.}
    \end{itemize}
    \end{answered}
  \item{What important facts link the scarcer self-peptide presentation by the protective MHC allele to presentation of a less diverse pool of self-peptide?}
    \begin{answered}
    \begin{itemize}
      \item{The subset of the human proteome presented as self-peptide during thymic selection is small relative to the total proteom}
      \item{For MHC to bind less self-peptide yet present just as diverse a subset to T cells during thymic selection, we'd need unusually many \say{collisions} for the protective allele (greater frequency of \textit{jointly} TCR- and MHC-compatible residues).}
      \item{Assuming independence between MHC allele and TCR compatibility, then, there's a vanishingly small probability of the protective allele \say{making up for} its poor self-peptide binding by binding equally many TCR-compatible peptides as MHC proteins from more broadly self-binding alleles.}
    \end{itemize}
    \end{answered}
  \item{Does this \say{protective allele} phenomenon appear to \textbf{generalize}? If so, how/to what?}
    \begin{answered}
    Yes, the HLA-B57 allele also appears to confer some protection agains \textbf{hepatitis C virus (HCV)}.
    \end{answered}
  \item{How/why does less broad self-peptide presentation by a MHC molecule yield a cognate TCR pool for that MHC that's more \say{tolerant of change,} or \say{robust to mutations} (i.e., better for HIV control)?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{\textit{Negative} selection} during thymocyte development eliminates overly self-reactive T cells (those binding to peptide:MHC complex too strongly, above some threshold).}
      \item{As more self-peptide is presented by a MHC, the probability rises for each compatible T cell that it will react too strongly with a complex and be eliminated.}
      \item{More self-peptide presentation by a MHC, then, tends to yield a cognate TCR pool enriched for weaker interactions.}
      \item{To still be positively selected despite individual residue's relative weak interaction with peptide, the TCRs for a broadly-self-presenting MHC tend to rely on more individual contacts with the peptide.}
      \item{Reliance by the TCR on more individual contacts with peptide means a given AA substitution in the peptide is more likely to ruin the recognition by the T cell.}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{Influenza Nature 2019 MHC classII receptor for novel bat IAV}
\begin{QandA}
  \item{What's flu's surface protein that it uses for entry? What's the usual receptor? What's the receptor for these two new bat IAVs (H17N10 and H18N11)?}
    \begin{answered}
    \begin{itemize}
      \item{Flu uses the \textit{hemagluttinin (HA)} protein to bind to a cell surface molecule.}
      \item{Normally, flu uses \textit{sialic acid,} an acidic sugar on cell surfaces, as its receptor; there's a connection between flu strain, host species, and sialic acid linkage preference (i.e., which bonds in the sialic acid the hemagglutinin prefers to bind).}
      \item{For these bat viruses, the receptor is not a sugar (like sialic acid), but rather a protein (MHC class II).}
    \end{itemize}
    \end{answered}
  \item{What's a a particularly striking difference between flu's typical receptor and the receptor for these bat viruses?}
    \begin{answered}
    Flu's normal receptor is an acidic sugar, but in these bat viruses it was thought previously, and shown here, to be a protein.
    \end{answered}
  \item{Which two methods were used to screen \textit{broadly} for candidate genes}
    \begin{answered}
    \begin{itemize}
      \item{\textit{Differential expression} contrasted gene expression from three cell cell lines susceptible to bat IAV with gene expression from three cell lines not susceptible to the virus, looking for genes up in all three susceptible cell lines relative to the non-susceptible lines. Candidates were filtered based on Gene Ontology search for plasma membrane association/function.}
      \item{\textit{Genome-wide CRISPR screening} was used transfecting \textit{Cas9}-expressing cells with lentiviral vectors carrying a bunch of guide RNAs, and then cells were selected for resistance to infection by VSV-H18 (VSV expressing one of the bat flu hemagluttinins). Then analysis showed which guide RNAs were in those cells (so which genes were knocked down).}
    \end{itemize}
    \end{answered}
  \item{What method was used to \textit{hone in on the target} gene(s)?}
    \begin{answered}
    With the genes up in all three susceptible cell lines (relative to non-susceptible) in hand, the investigators used \textbf{RNAi} (specifically, with \textbf{siRNA}) directed against the target genes.
    \end{answered}
  \item{In what main ways was it demonstrated definitively that MHC class II is the receptor for these new bat flu viruses?}
    \begin{answered}
    \begin{itemize}
      \item{Genetic knockout of MHC class II is protective against entry of H18 VLPs (controlled by H1N1 VLPs still showing ability to enter).}
      \item{siRNA knockdown (maybe) of MHC class II impedes H18 VLP entry.}
      \item{MHC class II expression from lentiviral vectors restores susceptibility to entry by H18 VLPs.}
    \end{itemize}
    \end{answered}
  \item{In what main ways was cell the link between the receptor both strengthened and generalized?}
    \begin{answered}
    \begin{itemize}
      \item{Many different species HLA-DR molecules confer susceptibility on HEK293 cells (express both chains for that MHC molecule) in the HEK293 cells and infect with VSV or VLPs with the bat flu hemagluttinins H18.}
      \item{Different MHC alleles also confer susceptibility (e.g., HLA-DRQ and HLA-DRP).}
    \end{itemize}
    \end{answered}
  \item{\textbf{Cross-species transmissibility}: What were the main findings about susceptibility in \textit{mice} to the actual bat viruses?}
    \begin{answered}
    \begin{itemize}
      \item{It appears that \textit{mice} are indeed susceptibile, but virus replication is \textit{confined to the \textbf{upper respiratory tract}}, and really only to a region of the \textbf{epithelium}.}
      \item{MHC class II deficiency is protective in mice.}
      \item{The link between MHC class II and susceptibility is strengthened by ablating MHC class II specifically on endothelial cells in the upper respiratory tract (basically the only replicatibe location) and demonstrating lack of plaque formation (i.e., no antigen-presenting cells still have MHC class II, but virus doesn't replicate).}
    \end{itemize}
    \end{answered}
\end{QandA}
\section{L08 Antibody Diversity Generation I}
\begin{QandA}
  \item{What were the 3 main phenomena about antibodies / antigen receptors that were known early in the history of immunology?}
    \begin{answered}
    \begin{itemize}
      \item{\textit{Strength in numbers}: It was known that the number of antibodies against a microbe/pathogen increases over primary challenge course, and is greater upon second encounter.}
      \item{\textit{Strength in strength}: It was known that the antibody response's affinity distribution \say{shifts right} in the course of a response.}
      \item{\textit{Isotype change / \say{class switch}}: It was known that the isotype/class of antibodies changed over the course of a primary response.}
    \end{itemize}
    \end{answered}
  \item{\textbf{\textit{Diversity} problem}: What's the \say{diversity problem}?}
    \begin{answered}
    \begin{itemize}
      \item{Antibodies are proteins, and it was thought that exactly one gene encoded each protein: \textit{\say{one protein, one gene.}}}
      \item{It was known that the number of human antibody specificities greatly outnumbered genes in the human genome.}
      \item{Therefore, there was a fundamental inconsistency between what was known about antibody specificities, the human genome, and protein coding.}
    \end{itemize}
    \end{answered}
  \item{What were two main early theories that attempted to explain the \textbf{\textit{diversity} problem}, and who developed each?}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{\textit{Side chain} theory} was developed by Paul Ehrlich.}
      \item{\textbf{\textit{Direct template} theory} was developed by Linus Pauling.}
    \end{itemize}
    \end{answered}
  \item{\textbf{\textit{Side chain} theory}: explain side chain theory and who developed it.}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Paul Ehrlich} developed the side chain theory and introduced it in \textbf{1900}.}
      \item{\textbf{\textit{Candidate} receptors}: Each antibody-producing cell begins with a variety of \say{candidate} receptors for binding antigen.}
      \item{\textbf{Selective \textit{relative abundance increase}}: The \say{candidate} receptor that binds specific antigen increases in abundance relative to other candidates.}
      \item{\textbf{Antibodies \say{fall off}}: Ehrlich proposed that the receptor candidate that bound antigen eventually increased in number and \say{fell off} the cell surface as soluble antibody.}
    \end{itemize}
    \end{answered}
  \item{\textbf{\textit{Side chain} theory}: explain side chain theory and who developed it.}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{Linus Pauling} developed the side chain theory in the early-/mid-1900s.}
      \item{\textbf{\textit{Prototype} receptor}: Unlike the multiple \say{candidates} of the side chain theory, each antibody-making cell has a single antigen receptor specificity \say{prototype.}}
      \item{\textbf{\textit{Conformational change} for affinity}: Pauling proposed that antigen binding facilitated affinity-increasing conformation change in the receptor.}
      \item{\textbf{\textit{Heritable} change}: Pauling proposed that the conformational change was heritable, i.e. present in subsequently produced antibodies.}
    \end{itemize}
    \end{answered}
  \item{\textit{Side chain theory flaws}: explain the three main flaws in side chain theory and what we now know that provides contradiction.}
    \begin{answered}
    \begin{itemize}
      \item{\textbf{\textit{Multiplicity}}: the side chain theory is wrong in its proposal that each relevant cell expresses multiple antigen receptor specificities; instead, monoallelic expression following successful lymphocyte development ensures singular specificity.}
      \item{\textbf{\textit{Solubility} mechanism}: rather than \say{falling off} the plasma membrane, the secreted receptor form (antibody) is made via \textit{alternative splicing} of the primary transcript, omitting the hydrophobic, membrane-spanning region that confers plasma membrane embedding/anchoring.}
      \item{\textbf{\textit{Proliferation} mechanism}: rather than the receptor molecule itself directly increasing in number, it's primarily \textit{cellular proliferation} that drives the increase in magnitude of the primary response and reaction to second encounter.}
    \end{itemize}
    \end{answered}
  \item{\textit{Direct template theory flaws}: explain the main flaw in direct template theory and what we now know that provides contradiction.}
    \begin{answered}
    \textbf{\textit{Affinity maturation} mechanism}: Rather than conformational changes in the initial antigen-binding receptor, it's mutations in progeny of that cell, via \textit{somatic hypermuation}, that give rise to affinity maturation.
    \end{answered}
\end{QandA}
\end{document}
